### PR TITLE
Hide export

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/lib/installation/proposal_runner.rb [hunk 1]
```diff
@@ -74,6 +74,9 @@ def run
         return :auto
       end
 
+      args = (Yast::WFM.Args || []).first || {}
+      @hide_export = args["hide_export"]
```
Comment (lslezak) [inline src/lib/installation/proposal_runner.rb]: NP: I'd probably named it `hide_export_button` to be clear what the export means...

**Comment by lslezak** [comment]
But still LGTM

File: src/lib/installation/proposal_runner.rb (hunk 1 above)
Comment (jreidinger) [inline src/lib/installation/proposal_runner.rb]: well, I mean it more generic like "hide export autoyast profile ability" it is not just about button ( even in fact it is only not adding button ).

Comment (lslezak) [inline src/lib/installation/proposal_runner.rb]: OK, then full LGTM :wink:

**Pull request merged**